docs/advanced_topics/images/renditions.rst

```diff
@@ -0,0 +1,28 @@
+.. _image_renditions:
+
+Generating renditions in Python
+=====================================
+
+Rendered versions of original images generated by the Wagtail ``{% image %}`` template tag are called "renditions,"
+and are stored as new image files in the site's ``[media]/images`` directory on the first invocation.
+
+Image renditions can also be generated dynamically from Python via the native ``get_rendition()`` method, e.g.:
+
+ .. code-block:: python
+
+    newimage = myimage.get_rendition('fill-300x150|jpegquality-60')
+
+Argument options are identical to arguments accepted by the ``{% image %}`` template tag, and should be separated
+with ``|``.
+
+The ``newimage`` object will have all of the usual properties of a Wagtail image (such as URL), so something like
```

Comment:
> The ``newimage`` object will have all of the usual properties of a Wagtail image

Except for ``title``, ``focal_point``, etc. Perhaps this sentence should be reworded